lib/Config.js

```diff
@@ -804,6 +804,12 @@ class Config extends EventEmitter {
         this.emit('rest-endpoints-update');
     }
 
+    setLocationConstraints(locationConstraints) {
```

Comment:
so to add a location constraint, pensieve has to send the whole lc back to s3? where does pensieve store the full location constraint file?

Reply:
Correct. My understanding is that it will be stored in a bucket used by zenko, and the retrieval of resources from there will managed by the Pensieve API.